go.mod

```diff
@@ -231,7 +231,6 @@ replace (
 	github.com/CosmWasm/wasmd => github.com/classic-terra/wasmd v0.46.0-classic.2
 	// use cometbft
 	github.com/cometbft/cometbft-db => github.com/cometbft/cometbft-db v0.8.0
-	github.com/cosmos/ibc-go/v7 => github.com/classic-terra/ibc-go/v7 v7.4.0-terra
```

Comment:
I think we should bump to the latest patch [7.4.1](https://github.com/cosmos/ibc-go/blob/v7.4.1/CHANGELOG.md) first and bump the minor version later.